path-manager/test/services/PathValidatorTest.scala

```diff
@@ -32,5 +44,9 @@ class PathValidatorTest extends PlaySpec {
     "reject paths with a combination of single quotes and white space" in {
       PathValidator.isValid("path/to/'something' else-news") must be(false)
     }
+
+    "reject a path with a hyphen after a slash" in {
+      PathValidator.isValid("path/to/-something") must be(false)
```

Comment:
Actually, thinking about it, if we didn't force this previously, are we going to break existing content?

Reply:
Published content won't get their paths revalidated, so are safe.

Draft content will go through this process, which should be fine as Composer restricts this pattern client-side already.